src/app/functions.js

```diff
@@ -0,0 +1,201 @@
+import i18n from '../i18n'
+
+const sumAll = numbers => {
+  let points = 0
+  numbers.forEach(number => {
+    points = points + number
+  })
+  return points
+}
+
+export const isThreeOfKind = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  if (isFullHouse(numbers)) return true
+  let sameNumbers = 0
+  const foundSameNumbers = []
+  sortedArray.reduce((a, b) => {
+    if (a === b) {
+      foundSameNumbers.push(b)
+      sameNumbers = sameNumbers + 1
+    }
+    return b
+  })
+  return sameNumbers === 2 && foundSameNumbers[0] === foundSameNumbers[1]
```

Comment:
For me the readability of the reduce function is bad in general.
It takes some time to get into it.

Small bug, these tests are failing:
```
expect (isThreeOfKind([1, 1, 1, 1, 4])).toBe(true)
expect (isThreeOfKind([1, 1, 1, 1, 1])).toBe(true)
```